### PR TITLE
Header headline merge to main

### PR DESCRIPTION
**Pull request by Thinus01**
HTML with the core structure of the mobile portfolio
CSS with visual styling for the mobile portfolio
CSS & HTML Linters added
Readme and license included

### CODE REVIEW

**Review by Kamba56 — COMMENTED** [review]
Hi <STUDENT_MENTION>, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a professional readme. Please add a professional readme using the template provided by microverse.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
```
Comment (HAMMAS-SALEEM) [inline .github/workflows/linters.yml:4]: ### Your Readme File
readme


- [ ] You didn't add a descriptive readme file as requested by the previous reviewer. Kindly make your readme file descriptive using [microverse template](https://raw.githubusercontent.com/microverseinc/readme-template/master/README.md) 😊

**Review by HAMMAS-SALEEM — COMMENTED** [review]
# Invalid Review Request 🔴

## Hey @Thinus01 :wave:

## Highlights :1st_place_medal:

:heavy_check_mark: Descriptive PR Title
✔️ Following Github flow
:heavy_check_mark: Following some requirements

Good job on implementing some of the requirements, But unfortunately i cannot proceed review on your pull request. there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/workflows/linters.yml (hunk 1 above)
Comment (HAMMAS-SALEEM) [inline .github/workflows/linters.yml:4]: - [ ] Great work on adding a readme file but this time you have submitted this project on the wrong link in your dashboard. This project is supposed to the 2nd project but you have asked for a review from the third project link. Kindly submit this project from the second project link of this week 👍

**Review by HAMMAS-SALEEM — COMMENTED** [review]
# Invalid Review Request 🔴

## Hey @Thinus01 :wave:

## Highlights :1st_place_medal:

:heavy_check_mark: Descriptive PR Title
✔️ Following Github flow
:heavy_check_mark: Following some requirements

Good job on implementing some of the requirements, But unfortunately i cannot proceed review on your pull request. there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @Thinus01, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] All linters are passing ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Good commit history ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇 👇 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @Thinus01, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+header {
+  margin: 0 auto;
+  height: 48px;
+  padding: 0 24px;
+}
```
Comment (PinkMoon25) [inline style.css:33]: [OPTIONAL] Your header is sticking to the top see :point_down: 
image

I think doing this :point_down: will fix it
```
header {
  padding:  24px;
}
```

**Review by PinkMoon25 — APPROVED** [review]
Hi @Thinus01 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
Excellent work implementing all the required changes, your project looks professional now :clap: 



## Highlights
Linters are passing :heavy_check_mark: 
Correct github flow :heavy_check_mark: 
Follows figma design :heavy_check_mark: 
Used Flexbox for layout :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Rana-Mudassir** [comment]
Everything is fine. It would be better if you add some comments in your code, so it becomes more understandable for the other collaborators. 
Cheers and Happy coding!👏👏👏

**Comment by chigozie1803** [comment]
you have done great job . I think it would be good to always make sure you delete every spaces in your code, for best practice.

**Pull request merged**